src/stories/lift.stories.tsx

```diff
@@ -0,0 +1,250 @@
+import React from 'react';
+import { Divider, Typography } from '@material-ui/core';
+
+import LiftComponent from './lift-component';
+
+export default { 
+    title: 'Lift',
+};
+
+const styles = {
+    modeInfoPanel: {
+        padding: '2rem'
+    },
+    modeInfoItem: {
+        display: 'flex',
+        justifyContent: 'space-between',
+        padding: '0.5rem'
+    }
+};
+
+const lift = {
+    depth: 2.5,
+    doors: [
+        {
+            door_type: 1, 
+            motion_direction: 1, 
+            motion_range: 0, 
+            name: 'lift1_front_door',
+            v1_x: 8.8,
+            v1_y: -2.3,
+            v2_x: 7.7,
+            v2_y: -4.5
+        },
+    ],
+    levels: ['L1', 'L2', 'L3', 'L4'],
+    name: 'Lift1',
+    ref_x: 7.1,
+    ref_y: -2.8,
+    ref_yaw: 0.5,
+    wall_graph: {
+        edges: [],
+        name: "wallgraph",
+        params: [],
+        vertices: [],
+    },
+    width: 2.5
+};
+
+const liftStates = {
+    available_floors: ["L1", "L2", "L3", "L4"],
+    available_modes: new Uint8Array(0),
+    current_floor: "L1",
+    current_mode: 0,
+    destination_floor: "L1",
+    door_state: 0,
+    lift_name: "Lift1",
+    lift_time: {sec: 0, nanosec: 0},
+    motion_state: 0,
+    session_id: "",
+}
+
+const AGVState = {
+    ... liftStates,
+    current_mode: 2,
+    door_state: 2,
```

Comment:
Can we use romi-js constants instead of literals? See https://osrf.github.io/romi-js-core-interfaces/classes/liftstate.html#door_closed.

Reply:
Yup sure. Thanks for pointing that out